packages/synapse-react-client/src/components/JsonSchemaForm/widgets/SelectWidget.tsx

```diff
@@ -97,7 +103,7 @@ export const SelectWidget: Widget = (props: SelectWidgetProps) => {
         const { inputValue } = params
         // Suggest the creation of a new value
         const isExisting = options.some(option => inputValue === option.label)
-        if (inputValue !== '' && !isExisting) {
+        if (allowFreeSolo && inputValue !== '' && !isExisting) {
           filtered.push({
             value: inputValue,
             label: `Set to custom value "${inputValue}"`,
```

Comment:
Label could be something like this, if we didn't want to refer to a user specified entry as a "custom value":
Use "${inputValue}" (not from list)

Reply:
I think the only current usage of this feature is annotations, but if we ever use it in another context totally agree that we can make this language more generic (or make it a parameter).